### DISCUSSION TITLE
Stats query : Seurat Average expression and Cell count output

### DISCUSSION THREAD

**Discussion by rasing02** [satijalab/seurat#9478] [General]
Hi,
For longitudinal studies (with variables like time, sex, treatment, etc.), is it okay to perform the following stats:
1. Use average expression output for all replicates and perform Kruskal Wallis or 2-way ANOVA (for treatment or sex effect) to see how gene expression changes over time in a specific cluster.
2. Convert the cell count of cluster 1 into the percentage of cells in cluster 1 for each replicate and then perform Kruskal Wallis or 2-way ANOVA (for treatment or sex effect) to see how the percentage of cells in a particular cluster changes over time in a specific cell type.
3. Please recommend a manuscript that includes such longitudinal comparisons.

Thank you,
Best,
R

**Comment by samuel-marsh** [comment]
Hi,

I would recommend using statistical methods designed to for differential expression and differential abundance rather than applying a “traditional” test.  

Best,
Sam

**Reply by rasing02** [reply]
Thank you Samuel. Can you provide more information on the differential abundance test in Seurat? Is it similar to test mentioned in this link https://github.com/phipsonlab/speckle
With log2FC, we can see shifts in reference to one comparison (Time X vs Y). I wanted to find a way to see longitudinal changes Time X, Y, Z, etc.  I appreciate your input.
